packages/@coorpacademy-components/src/template/skill-detail/all-courses.js

```diff
@@ -169,8 +166,7 @@ AllCourses.contextTypes = {
 };
 
 AllCourses.propTypes = {
-  courses: PropTypes.shape(CardsGrid.propTypes),
-  totalCourses: PropTypes.number,
+  content: PropTypes.shape(CardsGrid.propTypes),
```

Comment:
As the props changes, we will need to adjust for both skill/playlist detail page also or they will break